src/main/features/response/form/models/whenDidYouPay.ts

```diff
@@ -0,0 +1,56 @@
+import { IsDefined, MaxLength, ValidateNested } from 'class-validator'
+import { IsPastDate } from 'forms/validation/validators/datePastConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { IsValidYearFormat } from 'app/forms/validation/validators/isValidYearFormat'
+import { MomentFactory } from 'common/momentFactory'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+import { MomentFormatter } from 'app/utils/momentFormatter'
+
+export class ValidationErrors {
+  static readonly EXPLANATION_REQUIRED: string = 'Explain how did you pay the amount claimed'
+  static readonly DATE_REQUIRED: string = 'Enter a date'
+  static readonly DATE_INVALID_YEAR: string = 'Enter a 4 digit year'
```

Comment:
this value is common to other places please use the global validation errors message.
REQ

Reply:
The same here, great candidate for common validation message const however I would recommend another PR. Decision on @timja-kainos side.

Reply:
extract here, separate PR to update other places please